dask_kubernetes/core.py

```diff
@@ -28,6 +29,16 @@
 logger = logging.getLogger(__name__)
 
 
+class PodStatus(Enum):
+    pending = 'Pending'
+    running = 'Running'
+    succeeded = 'Succeeded'
+    failed = 'Failed'
+    unknown = 'Unknown'
+    completed = 'Completed'
+    crashloopbackoff = 'CrashLoopBackOff'
```

Comment:
I have mixed thoughts about specifying things like this.  I imagine it's a matter of style.  I've personally always preferred using raw values directly rather than providing an abstraction layer between them.  This enum approach might be better as people use things like type systems and nicer editors though.  Maybe it's time for my style to change.

Reply:
After chatting with @mrocklin we came to the conclusion that for small codebases and readability we should switch to global variables rather than introduce ENUMs